bin/bnf_to_cnf/bnf_to_cnf/driver.py

```diff
@@ -9,8 +11,7 @@
 parser = argparse.ArgumentParser(description="Convert BNF grammar to CNF")
 parser.add_argument(
     "file",
-    nargs=1,
-    type=str,
+    type=Path,
```

Comment:
Why remove the `nargs=1` parameter?

Reply:
Ah, I guess that makes this a single use argument implicitly, and removes the need to use `args.file[0]` when accessing the value.